src/resilience/openMP/OpenMPResParallel.hpp

```diff
@@ -48,7 +48,7 @@
 #include <omp.h>
 #include <iostream>
 
-#include <OpenMP/Kokkos_OpenMP_Exec.hpp>
+#include <OpenMP/Kokkos_OpenMP_Instance.hpp>
```

Comment:
These are private kokkos headers do not include them

Reply:
Sorry for the late reply, I was out for a while due to illness. They have been removed, and I'm in the process of purging all private headers now that I have the list.